cookbook/deployment/fortrabbit.rst

```diff
@@ -197,8 +197,8 @@ Deploying to fortrabbit
 It is assumed that your codebase is under version-control with Git and dependencies
 are managed with Composer (locally).
 
-Every time you push to fortrabbit composer install runs before your code gets 
-deployed. To finetune the deployment behavior put a `fortrabbit.yml`_. deployment 
+Every time you push to fortrabbit composer install runs before your code gets
+deployed. To finetune the deployment behavior put a `fortrabbit.yml`_. deployment
```

Comment:
The best probably would be to make the clean up on the `2.3` branch to reduce the potential for merge conflicts.